src/gsy_e/models/market/future.py

```diff
@@ -134,14 +134,19 @@ def orders_per_slot(self) -> Dict[str, Dict]:
         for time_slot, bids_list in self.slot_bid_mapping.items():
             time_slot = time_slot.format(DATE_TIME_FORMAT)
             if time_slot not in orders_dict:
-                orders_dict[time_slot] = {"bids": [], "offers": []}
+                orders_dict[time_slot] = {
```

Comment:
Followup to the former suggestion, you could add the market.info (or "market_type_name" as you have here, the logic is similar) as a separate key together with the timeslots. That way you can avoid duplicating the `"market_type_name": "Test Future Market"` for all future time slots, and have this information only once.